client/src/pages/Component/CodeEditor/CodeEditor.tsx

```diff
@@ -24,6 +31,50 @@ const CodeEditor = ({
     },
   ];
 
+  if (programName != '') {
+    const loadCode = () => {
+      axios
+        .get(`${SERVER_URL}/api/retrieveFile`, {
+          params: {
+            username: PLACEHOLDER_USERNAME,
+            workspace: workspaceName,
+            filename: programName,
+          },
+        })
+        .then((response) => {
+          if (response.data.hasOwnProperty('error')) {
+            console.log('ERROR: ', +response.data.error);
+            return '';
+          }
```

Comment:
Handling errors is easier with `.catch()` to catch when the server responds wit ha status code to indicate an error.
```ts
axios.get(...).then(...).catch((error) => {
  console.log(error);
  return '';
})
```